### PR TITLE
karthik_subscription_30Dec2019

### PR DESCRIPTION
**Pull request by karthik0017**

### CODE REVIEW

File: .idea/modules.xml [hunk 1]
```diff
@@ -3,6 +3,7 @@
   <component name="ProjectModuleManager">
     <modules>
       <module fileurl="file://$PROJECT_DIR$/.idea/nest-web.iml" filepath="$PROJECT_DIR$/.idea/nest-web.iml" />
+      <module fileurl="file://$PROJECT_DIR$/.idea/nest-web.iml" filepath="$PROJECT_DIR$/.idea/nest-web.iml" />
```
Comment (srans-admin) [inline .idea/modules.xml]: Why this file required?

What is the PR name, next time raise PR subject as karthik_subscription_30Dec2019

File: src/app/app-routing.module.ts [hunk 2]
```diff
@@ -101,6 +109,11 @@ const routes: Routes = [
   { path: 'complaintupdate/:id', component: UpdateComplaintComponent},
   { path: 'compaintdetails/:id', component: ComplaintDetailsComponent},
 
+ { path: 'categorys', component: CategoryListComponent },
```
Comment (srans-admin) [inline src/app/app-routing.module.ts]: correct the spellings

File: src/app/app-routing.module.ts [hunk 3]
```diff
@@ -101,6 +109,11 @@ const routes: Routes = [
...
+  { path: 'categorys/add', component: CreateCategoryComponent },
```
Comment (srans-admin) [inline src/app/app-routing.module.ts]: I dont understand Why categories are comming now. Those should be in expenses related PR right

Comment (karthik0017) [reply src/app/app-routing.module.ts]: I forgot to add this file sir

**Pull request merged**